PIL/ImageDraw.py

```diff
@@ -237,10 +236,17 @@ def text(self, xy, text, fill=None, font=None, anchor=None,
                     mask = font.getmask(text, self.fontmode)
                 except TypeError:
                     mask = font.getmask(text)
+            if outline is not None:
+                color, _ = self._getink(outline)
+                for offset in range(0, 2):
+                    tmp_xy = list(xy)
+                    for difference in [-1, 1]:
+                        tmp_xy[offset] = xy[offset] + difference
+                        self.draw.draw_bitmap(tmp_xy, mask, color)
             self.draw.draw_bitmap(xy, mask, ink)
 
     def multiline_text(self, xy, text, fill=None, font=None, anchor=None,
-                       spacing=4, align="left"):
+                       outline=None, spacing=4, align="left"):
```

Comment:
Put `outline` at the end, because changing the order of parameters can break existing user code.